ksql-cli/src/test/java/io/confluent/ksql/cli/SslFunctionalTest.java

```diff
@@ -109,15 +110,20 @@ public static void classSetUp() {
 
   @Before
   public void setUp() {
-    clientProps = Collections.emptyMap();
+    clientProps = new HashMap<>();
+    clientProps.put(KsqlClient.DISABLE_HOSTNAME_VERIFICATION_PROP_NAME, "true");
```

Comment:
nit: unnecessary in light of the additions to `givenTrustStoreConfigured()` and `givenClientConfguredWithoutTruststore()`.

Reply:
ack